CHANGES.rst

```diff
@@ -1,6 +1,8 @@
 1.5.1 (Unreleased)
 ------------------
 
+- Do not apply pipeline-default WCSs from astrometry database as new WCS. [#96]
+
```

Comment:
Is this going to DMS now? If not should it go into 1.5.2?

Reply:
We do not have any reason now for this change to go to DMS now (something I was not sure of when I first submitted this PR), so I changed this to a comment on 1.5.2 in commit 6c60cf6.